lua/telescope/_extensions/file_browser.lua

```diff
@@ -148,6 +143,18 @@ local file_browser = function(opts)
   fb_picker(popts)
 end
 
+local fb_setup = function(opts)
+  -- TODO maybe merge other keys as well from telescope.config
+  pconf.mappings = vim.tbl_deep_extend("force", pconf.mappings, require("telescope.config").values.mappings)
+  pconf = vim.tbl_deep_extend("force", pconf, opts)
+
+  if pconf.hijack_netrw then
+    vim.cmd "silent! autocmd! FileExplorer *"
+    vim.cmd "autocmd VimEnter * ++once silent! autocmd! FileExplorer *"
```

Comment:
So how is netrw supposed to work here? IIRC it only opens when you open a folder, right? e.g. `nvim .` or `nvim some_folder` rather than `nvim`.

I don't think file browser should always be opened on `VimEnter`.

Reply:
You're right, I'll add tweak this as well